storage/spanstore/utils.go

```diff
@@ -0,0 +1,17 @@
+package spanstore
+
+import (
+	"hash"
+	"math"
+)
+
+// HashBytes returns the uint64 hash value of bytes slice
+func HashBytes(h hash.Hash64, bytes []byte) uint64 {
+	h.Reset()
```

Comment:
this is not thread-safe

Reply:
you may want to use a sync.Pool to avoid using lock or allocating too many hash objects.

Reply:
also, do you have to call `Reset` every time? According to the wiki: `Start with an initial hash value of FNV offset basis. For each byte in the input, multiply hash by the FNV prime, then XOR it with the byte from the input.` so it looks like you can keep writing rather than resetting.